plugins/azure-devops/README.md

```diff
@@ -313,3 +313,62 @@ To get the README component working you'll need to do the following two steps:
 - You'll need to add the `EntitySwitch.Case` above from step 2 to all the entity sections you want to see Readme in. For example if you wanted to see Readme when looking at Website entities then you would need to add this to the `websiteEntityPage` section.
 - The `if` prop is optional on the `EntitySwitch.Case`, you can remove it if you always want to see the tab even if the entity being viewed does not have the needed annotation
 - The `maxHeight` property on the `EntityAzureReadmeCard` will set the maximum screen size you would like to see, if not set it will default to 100%
+
+## Permission Framework
+
+Azure DevOps plugin is now supporting the permission framework for PRs, GitTags, Pipelines & ReadMe.
+
+To enable the permission on the legacy backend system add the below config in `packages/backend/src/plugins/azure-devops.ts`.
```

Comment:
Looks like we also need to add the common package:

```
# From your Backstage root directory
yarn add --cwd packages/backend @backstage/plugin-azure-devops-common
```

Reply:
`@backstage/plugin-azure-devops-common`  is already part of `@backstage/plugin-azure-devops-backend` dependencies

https://github.com/deepan10/backstage/blob/41b0f2cd3be889ce079ad8bd996577889f08db70/plugins/azure-devops-backend/package.json#L37-L39

I believe the adopters no need to add `@backstage/plugin-azure-devops-common` explictly in `packages/backend`

Reply:
Yes, it needs to be added. If it's not there they will get an error when trying to import the permissions you added. When I did this last review I stepped though setting up the policy myself to be able to test and ran into this problem. It's always good to manually test everything.